tcms/telemetry/static/telemetry/js/testing/utils.js

```diff
@@ -1,10 +1,19 @@
 function loadInitialProduct(callback = () => {}) {
     jsonRPC('Product.filter', {}, data => {
-        updateSelect(data, '#id_product', 'id', 'name');
+        updateSelect(data, '#id_product', 'id', 'name', null);
         callback();
     });
 }
 
 function showOnlyRoundNumbers(number) {
     return number % 1 === 0 ? number : '';
 }
+
+// Close multiselect list when selecting an item
+// Iterate over all dropdown lists
+$("select:not([size])").each(function() {
```

Comment:
Can you explain the selector here? Why not searching for `select:[multiple]` ?

Reply:
`select:[multiple]` also did not seem to work. this selector was based on what i read [here](https://stackoverflow.com/a/1664728)

Reply:
The correct syntax is `select[multiple]` -> https://api.jquery.com/attribute-equals-selector/

Reply:
Fixed in https://github.com/kiwitcms/Kiwi/pull/2441/files